src/mozillavpn.cpp

```diff
@@ -927,6 +920,10 @@ void MozillaVPN::mainWindowLoaded() {
 void MozillaVPN::onboardingCompleted() {
   SettingsHolder* settingsHolder = SettingsHolder::instance();
 
+  //Set glean based on data collection checkbox checked state
+  QObject* dataCollectionCheckbox = InspectorUtils::queryObject("//dataCollectionCheckBox");
```

Comment:
This feels fragile. What if the user restart the application while in the middle of onboarding? Won't they directly go to the last step? What is the state of the data collection checkbox then? It might be safer to use a setting for the state of this checkbox.